tests/routes/portfolios/test_admin.py

```diff
@@ -26,3 +27,103 @@ def test_member_table_access(client, user_session):
     user_session(rando)
     view_resp = client.get(url)
     assert "<select" not in view_resp.data.decode()
+
+
+def test_permission_set_has_changed_with_changes():
+    old_set = [
+        "edit_portfolio_application_management",
+        "view_portfolio_application_management",
+    ]
+    new_set = ["view_portfolio_application_management"]
+    assert permission_set_has_changed(old_set, new_set)
+
+    old_set = ["view_funding"]
+    new_set = ["edit_funding"]
+    assert permission_set_has_changed(old_set, new_set)
+
+
+def test_permission_set_has_changed_without_changes():
+    old_set = [
+        "edit_portfolio_application_management",
+        "view_portfolio_application_management",
+    ]
+    new_set = ["edit_portfolio_application_management"]
+    assert not permission_set_has_changed(old_set, new_set)
+
+    old_set = ["view_funding"]
+    new_set = ["view_funding"]
+    assert not permission_set_has_changed(old_set, new_set)
+
+
+def test_update_member_permissions(client, user_session):
+    portfolio = PortfolioFactory.create()
+    rando = UserFactory.create()
+    rando_pf_role = PortfolioRoleFactory.create(
+        user=rando,
+        portfolio=portfolio,
+        permission_sets=[PermissionSets.get(PermissionSets.VIEW_PORTFOLIO_ADMIN)],
+    )
+
+    user = UserFactory.create()
+    PortfolioRoleFactory.create(
+        user=user,
+        portfolio=portfolio,
+        permission_sets=[PermissionSets.get(PermissionSets.EDIT_PORTFOLIO_ADMIN)],
```

Comment:
```suggestion
        permission_sets=PermissionSets.get_many(
            [PermissionSets.EDIT_PORTFOLIO_ADMIN, PermissionSets.VIEW_PORTFOLIO_ADMIN]
        ),
```
This should fix this test. You're getting an error because the initial POST is successful, but then the user isn't authorized to view the admin page when they're redirected.